Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -274,6 +274,8 @@
     <string name="preferences_settings">Settings</string>
     <string name="preferences_game_settings">Game Settings</string>
     <string name="preferences_extensions">Extension Bindings</string>
+    <string name="game_ini_junk_title">Junk Data Found</string>
+    <string name="game_ini_junk_question">The settings file for this game contains junk data created by an old version of Dolphin. Would you like to fix this by deleting the settings file for this game? This cannot be undone.</string>
```

Comment:
...then again, I wrote this comment before we established that this is just for _editing_ settings (and not during boot as I initially thought); so it might be obvious enough that user-specific settings would be dropped in this case.

Reply:
Yeah, I can't really think of a way to make the explanation clearer that isn't just stating the obvious.